src/libraries/System.Text.Encoding.CodePages/src/Data/Tools/EncodingDataGenerator.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Will this be built on every vertical?

Reply:
@safern thanks for bringing the issue. I have checked the bin logs and I am seeing the `_allSrc` is not including the tools folder to build.So, I think we are fine here.

Reply:
Great, thanks for double checking 😄